libs/components/action-bars/src/lib/modules/summary-action-bar/summary-action-bar.component.scss

```diff
@@ -69,66 +127,84 @@
-    padding: $sky-padding;
+    padding: var(
+      --sky-override-summary-action-bar-summary-body-collapsible-padding,
+      var(--sky-space-inset-pillarbox-2_3-top-m)
+        var(--sky-space-inset-pillarbox-2_3-right-m)
+        var(--sky-space-inset-pillarbox-2_3-bottom-m)
+        var(--sky-space-inset-pillarbox-2_3-left-m)
+    );
   }
 
   .sky-summary-action-bar-details-expand,
   .sky-summary-action-bar-details-collapse {
     display: block;
   }
-}
 
-@include defaultMixins.sky-theme-modern {
-  .sky-summary-action-bar-actions {
-    padding: $sky-theme-modern-space-lg $sky-theme-modern-space-xl;
-  }
-
-  .sky-summary-action-bar-summary {
-    border: none;
+  .sky-summary-action-bar-details-expand {
+    margin-left: auto;
+    margin-right: var(
+      --sky-override-summary-action-bar-details-expand-margin-right,
+      var(--sky-space-inline-xl)
+    );
   }
 
-  .sky-summary-action-bar-summary-body {
-    padding-left: $sky-theme-modern-space-xl;
+  .sky-summary-action-bar-details-collapse {
+    margin-left: var(
+      --sky-override-summary-action-bar-details-collapse-margin-left,
+      var(--sky-space-gap-text_action-l)
+    );
+    margin-right: var(
+      --sky-override-summary-action-bar-details-collapse-margin-right,
+      calc(
+        var(--sky-space-inline-xl) - var(
+            --sky-space-inset-pillarbox-2_3-right-m
+          )
+      )
+    );
   }
 
-  .sky-summary-action-bar-summary-collapsible {
-    .sky-summary-action-bar-actions {
-      padding: $sky-theme-modern-padding-even-lg;
-    }
-
-    .sky-summary-action-bar-summary-body {
-      padding: $sky-theme-modern-padding-even-lg;
-    }
+  .sky-summary-action-bar-actions {
+    padding: var(
+      --sky-override-summary-action-bar-summary-actions-collapsible-padding,
+      var(--sky-space-inset-pillarbox-2_3-top-m)
+        var(--sky-space-inset-pillarbox-2_3-right-m)
+        var(--sky-space-inset-pillarbox-2_3-bottom-m)
+        var(--sky-space-inset-pillarbox-2_3-left-m)
+    );
   }
+}
 
+@include defaultMixins.sky-theme-modern {
   .sky-summary-action-bar-summary-items {
-    margin-bottom: -$sky-theme-modern-margin-inline-sm;
+    // NOTE: This is to maintain backwards compatibility with v1 modern. We however, were removing extra space when key infos are not used.
+    // This is addressed by the `:has` selector below and this unscoped margin bottom adjustment should be removed when only v2 exists.
+    margin-bottom: var(
+      --sky-override-summary-action-bar-summary-items-margin-bottom,
+      0
+    );
+
+    &:has(::ng-deep sky-key-info:last-child) {
+      margin-bottom: calc(var(--sky-space-inline-s) * -1);
+    }
 
     ::ng-deep sky-key-info {
-      margin: 0 $sky-theme-modern-margin-inline-xl
-        $sky-theme-modern-margin-inline-sm 0;
+      margin: 0 var(--sky-space-inline-xl) var(--sky-space-inline-s) 0;
       display: inline-block;
     }
 
     ::ng-deep sky-key-info:last-child {
```

Comment:
I chatted about this `ng-deep` with Matt. We both feel that it is odd - but we don't know what all removing it might affect for consumers and so we decided to leave it for now.